spec/requests/graphql/mutations/users/log_in_spec.rb

```diff
@@ -18,7 +18,7 @@
       before do
         create(:user, :coach, email: email, password: password)
         allow(Authentication::JwtToken::CreateService)
-          .to receive(:call).and_return(Dry::Monads::Result::Success.new("token"))
+          .to receive(:call).and_return(Success("token"))
```

Comment:
@ManuSG8 
We can now use `Success()` and `Failure()` monads in our test layer without having to use the absolute module path.